account_banking_mandate/models/account_banking_mandate.py

```diff
@@ -176,8 +176,10 @@ def _check_valid_state(self):
     @api.model_create_multi
     def create(self, vals_list):
         for vals in vals_list:
-            unique_mandate_reference = vals.get("unique_mandate_reference", "/")
-            if unique_mandate_reference == "/":
+            if (
+                not vals.get("unique_mandate_reference")
+                or vals.get("unique_mandate_reference") == "/"
+            ):
```

Comment:
```suggestion
            if vals.get("unique_mandate_reference", "/") == "/":
```

Reply:
Sorry, but that's how it was previously and it didn't cover the use case of leaving the reference empty.

Reply:
Ah, OK, that's right. Then it should be:

```suggestion
            if (vals.get("unique_mandate_reference") or "/") == "/":
```